src/api/fraudnet.js

```diff
@@ -34,7 +35,7 @@ type FraudnetConfig = {|
 
 type LoadFraudnet = (FraudnetOptions) => ZalgoPromise<void>;
 
-export const loadFraudnet : Memoized<LoadFraudnet> = memoize(({ env, clientMetadataID, cspNonce, timeout = 1000 }) => {
+export const loadFraudnet : Memoized<LoadFraudnet> = memoize(({ env, clientMetadataID, cspNonce, timeout = 1000, fraudnetUrlParams }) => {
```

Comment:
Let's default this value to an empty object.
```suggestion
export const loadFraudnet : Memoized<LoadFraudnet> = memoize(({ env, clientMetadataID, cspNonce, timeout = 1000, queryStringParams = {} }) => {
```